yarn.lock

```diff
@@ -6285,20 +6285,20 @@ eslint-config-prettier@^4.3.0:
   dependencies:
     get-stdin "^6.0.0"
 
-eslint-config-react-app@^4.0.1:
-  version "4.0.1"
-  resolved "https://registry.yarnpkg.com/eslint-config-react-app/-/eslint-config-react-app-4.0.1.tgz#23fd0fd7ea89442ef1e733f66a7207674b23c8db"
-  integrity sha512-ZsaoXUIGsK8FCi/x4lT2bZR5mMkL/Kgj+Lnw690rbvvUr/uiwgFiD8FcfAhkCycm7Xte6O5lYz4EqMx2vX7jgw==
-  dependencies:
-    confusing-browser-globals "^1.0.7"
-
 eslint-config-react-app@^5.2.0:
   version "5.2.0"
   resolved "https://registry.yarnpkg.com/eslint-config-react-app/-/eslint-config-react-app-5.2.0.tgz#135110ba56a9e378f7acfe5f36e2ae76a2317899"
   integrity sha512-WrHjoGpKr1kLLiWDD81tme9jMM0hk5cMxasLSdyno6DdPt+IfLOrDJBVo6jN7tn4y1nzhs43TmUaZWO6Sf0blw==
   dependencies:
     confusing-browser-globals "^1.0.9"
 
+eslint-config-react-app@^5.2.1:
```

Comment:
Now there are two versions of `eslint-config-react-app`. I think the second one comes from react-scripts used by the demo. But it should be resolved to `v5.2.1`.

Is it possible to only keep this one and remove the `v5.2.0`?

Reply:
For what I have searched, you can't simply remove duplications.
The simplest solution is to change this PR to target `eslint-config-react-app@^5.2.0` that is enough.
**What do you think?**



**Side Note**: 
Looking at `yarn-deduplicate` there are a lot of duplicated packages version

Output of `yarn-deduplicate --list`:

`Package "@babel/runtime" wants ^7.0.0 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.1.2 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.3.1 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.3.4 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.4.0 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.4.4 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.4.5 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.5.5 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.6.2 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.6.3 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.7.2 and could get 7.10.1, but got 7.8.4`
`Package "@babel/runtime" wants ^7.7.4 and could get 7.10.1, but got 7.8.4`
`Package "eslint-config-react-app" wants ^5.2.0 and could get 5.2.1, but got 5.2.0`
`Package "minimist" wants ^1.1.1 and could get 1.2.5, but got 1.2.0`
`Package "minimist" wants ^1.1.3 and could get 1.2.5, but got 1.2.0`
`Package "minimist" wants ^1.2.0 and could get 1.2.5, but got 1.2.0`
`Package "mkdirp" wants 0.x and could get 0.5.5, but got 0.5.1`
`Package "mkdirp" wants ^0.5.0 and could get 0.5.5, but got 0.5.1`
`Package "mkdirp" wants ^0.5.1 and could get 0.5.5, but got 0.5.1`
`Package "mkdirp" wants ~0.5.0 and could get 0.5.5, but got 0.5.1`
`Package "mkdirp" wants ~0.5.1 and could get 0.5.5, but got 0.5.1`
`Package "react-transition-group" wants ^4.3.0 and could get 4.4.1, but got 4.3.0``
`Package "regenerator-runtime" wants ^0.13.2 and could get 0.13.5, but got 0.13.3`
`Package "regenerator-runtime" wants ^0.13.3 and could get 0.13.5, but got 0.13.3`